src/app/register/page.tsx

```
@@ -21,14 +24,42 @@ export default function RegisterPage() {
 		setFormData((prev) => ({ ...prev, [field]: value }));
 	};
 
-	const handleSubmit = (e: React.FormEvent) => {
+	const [isSubmitting, setIsSubmitting] = useState(false);
```

ESLint

[error] 'isSubmitting' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)